### PR TITLE
Auto 3033 fix incorrect nn path

### PR DESCRIPTION
**Pull request by MRo47**
In 

```cpp
void parseConfigFile(const std::string& path, std::shared_ptr<T> nn, std::shared_ptr<dai::node::ImageManip> imageManip) {
        using json = nlohmann::json;
        std::ifstream f(path);
        json data = json::parse(f);
        if(data.contains("model") && data.contains("nn_config")) {
            auto modelPath = getModelPath(data);
            declareAndLogParam("i_model_path", modelPath);
            if(!getParam<bool>("i_disable_resize")) {
                setImageManip(modelPath, imageManip);
            }
            ...
```
- At line `declareAndLogParam("i_model_path", modelPath);` above (in [original_file](https://github.com/botsandus/depthai-ros/blob/cf5d2aaee9117298ea1632c98ffd36a5d7d535ac/depthai_ros_driver/include/depthai_ros_driver/param_handlers/nn_param_handler.hpp#L107))
- The ros parameter is overridden with the path from nn_config file, line 3 below. This sometime might not be a path at all.
```cpp
// nn_config.json
{
  "model": {
    "model_name": "forklift_openvino_yolov8n_2022",
    "zoo": "path"
  },
  ...
```
- Since paths in the nn_config file cannot be hard coded but rather rely on the location of the shared install directory, it should be of lower precedence than a ros param (i_model_path) if its set.

## TLDR
- model path in nn_config file overrides ros param
- this solves the issue by giving a higher priority to ros param, i.e: if its set it will override the path in nn_config file

### CODE REVIEW

**Comment by MRo47** [comment]
got merged by vendor
